keps/sig-storage/1885-csi-migration-azurefile/kep.yaml

```diff
@@ -22,17 +22,18 @@ prr-approvers:
 replaces:
 
 # The target maturity stage in the current dev cycle for this KEP.
-stage: beta
+stage: ga
```

Comment:
this is failing validation - should be stable

Reply:
fixed